crates/nargo_cli/src/cli/init_cmd.rs

```diff
@@ -9,7 +9,11 @@ use std::path::PathBuf;
 
 /// Create a Noir project in the current directory.
 #[derive(Debug, Clone, Args)]
-pub(crate) struct InitCommand;
+pub(crate) struct InitCommand {
+    /// Name of the package (Defaults to current directory name)
```

Comment:
```suggestion
    /// Name of the package [default = current directory name]
```
Any thoughts on this format?

Reply:
Yeah, I prefer that to what I had originally. I've updated to match the format which we use in `nargo prove` however.